style.css

```diff
@@ -0,0 +1,805 @@
+* {
+  margin: 0;
+  padding: 0;
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body {
+  background: #1c1a19;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.7rem;
+  margin-bottom: 65px;
+  margin-right: 24px;
```

Comment:
please you can easily write one code for these codes, the less numbers of codes the good way of coding.